frontend/src/components/EntityBulkAdd.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div v-for="num in entityFormComponentsCountRange" :key="`ef-${num}`">
+    <EntityForm :ref="`entity-form-${num}`" :schema="schema"
```

Comment:
I must say I dislike having the "save all" button under each form, it looks strange. Maybe it could be just a single button at the bottom? In addition the UX would benefit from a (smooth) scroll once you click "add more", so that you don't need to scroll manually down each time.

Reply:
Agree for both, but unfortunately for the save all button additional refactoring should be made.. the entity form is used for bulk edit also and the behavior is the same there

Reply:
yeah... that happens if forms aren't done using v-model and handle the actions (with buttons) internally :disappointed:
I'd say if you decide for not doing any refactoring, leave the buttons as is rather than doing some crazy if/else tweaks in the form component.